src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -477,6 +503,17 @@ export default {
 			return 'span'
 		},
 
+		wrapperStyle() {
+			return [
+				this.style,
+				// CSS local variables for this component
+				{
+					'--icon-size': this.size + 'px',
+					'--icon-height': (this.type === TYPE_SWITCH ? 16 : this.size) + 'px',
+				},
+			].flat()
```

Comment:
`.flat()` is not needed in both here and class binding, it is covered by Vue

Reply:
:writing_hand: